object-construction-checker/tests/mustcall/MustCallChoiceOwningField.java

```diff
@@ -0,0 +1,28 @@
+// Based on a MustCallChoice scenario in Zookeeper.
+
+import org.checkerframework.checker.objectconstruction.qual.*;
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.checker.calledmethods.qual.*;
+import java.io.*;
+
+public @MustCall("shutdown") class MustCallChoiceOwningField {
```

Comment:
How is this testing `@MustCallChoice`?  I don't see that annotation in the test

Reply:
It's written in stub files.